circuits/kimchi/src/nolookup/constraints.rs

```diff
@@ -605,7 +605,17 @@ impl<F: FftField + SquareRootField> ConstraintSystem<F> {
             // check if wires are connected
             for col in 0..PERMUTS {
                 let wire = gate.wires[col];
-                assert!(wire.col < PERMUTS);
+
+                if wire.col < PERMUTS {
```

Comment:
This inequality is the opposite of what it should be

Reply:
this is fixed in a later commit :o